docs/api/permission.rst

```diff
@@ -47,8 +47,9 @@ The :data:`zope.security.checker.CheckerPublic` permission always exists:
 
    >>> ids = list(allPermissions(None))
    >>> ids.sort()
-   >>> ids
-   ['x', 'y']
+   >>> for i in ids: print(i)
```

Comment:
Might as well rewrite the above to `>>> ids = sorted(allPermissions(None))` while at it.

Reply:
OK.